### PR TITLE
[BACKLOG-30151] data lineage logging for pentaho server logs

### PR DESCRIPTION
**Pull request by NJtwentyone**

### CODE REVIEW

**Comment by NJtwentyone** [comment]
Ignore sonarQube results. I added a functions in BaseClass BaseRuntimeExtensionPoint which have unit tests for. I can't test or marginal utility in use of log statements in existing code.

**Comment by NJtwentyone** [comment]
please review @pentaho/r2d2

File: core/src/main/resources/com/pentaho/metaverse/messages/messages.properties [hunk 1]
```diff
@@ -73,17 +73,15 @@ INFO.JobOrTrans.Status_1=DRAFT
...
+INFO.JobAnalyzeStarting=Data Lineage Analyzer for Job {0} - starting
+INFO.JobAnalyzeFinished=Data Lineage Analyzer for Job {0} - finished
 
 DEBUG.Locator.StopScan=Locator type {0}: stopScan()
 DEBUG.Locator.StartScan=Locator type {0}: startScan()
 DEBUG.Lineage.noProfileOutputStream=No profile output stream associated with this LineageWriter
 DEBUG.Lineage.noGraphOutputStream=No graph output stream associated with this LineageWriter
 #DEBUG.FallingBackToFirstSetOfInputFields=Step to use to determine which step to use for previous fields is unknown, choosing {0}.
-
-TITLE.Metaverse=Metaverse
```
Comment (cseverino789) [inline core/src/main/resources/com/pentaho/metaverse/messages/messages.properties]: Was the Title.Metaverse removed based on not being used? Just want to double check it was not removed unintentionally

Comment (NJtwentyone) [reply core/src/main/resources/com/pentaho/metaverse/messages/messages.properties]: It's no longer used. I removed it during this PR.

**Pull request merged**